eth/protocol.go

```diff
@@ -17,17 +17,28 @@
 package eth
 
 import (
+	"fmt"
+	"io"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// Constants to match up protocol versions and messages
+const (
+	eth61 = 61
+	eth62 = 62
+	eth63 = 63
+	eth64 = 64
```

Comment:
Originally I also had only numbers. The reason I added the constants is that this way if I want to kill off a version, I can just delete one line, and the entire code will get full of compile errors, so I can be sure that all traces of some protocol are deleted. I know this is not fool proof, but hopefully it's better that having some magic number leftovers in the code.